cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
Right. The server part only checks hostname when filtering. But the code logic is actually from [here](https://github.com/jinuxstyle/swarmkit/blob/4399144e7f68d9fad4c9dea6c7575cd562f3e1e2/cmd/swarmctl/node/list.go#L45) in cmd/swarmctl/node/list.go. Should I change them both to keep them consistent or just leave them as they are? I guess the intention of the code is that the Annotations.Name from Spec is always the first place to look for a name, like the names for services, etc.